cloud-azure/src/main/java/com/sequenceiq/cloudbreak/cloud/azure/AzureEncryptionResources.java

```diff
@@ -34,14 +35,16 @@
 
 @Service
 public class AzureEncryptionResources implements EncryptionResources {
+
+    @VisibleForTesting
+    static final Pattern DISK_ENCRYPTION_SET_RESOURCE_GROUP = Pattern.compile(".*resourceGroups/([^/]+)/providers.*");
```

Comment:
why visible for testing?

Reply:
The change is to correct the pattern, using this pattern in test will ensure that future changes to pattern would be tested appropriately.